docs/sources/operators-guide/architecture/components/ruler/index.md

```diff
@@ -14,6 +14,17 @@ Each tenant has a set of recording and alerting rules and can group those rules
 
 ![Architecture of Grafana Mimir's ruler component](ruler.svg)
 
+## Operational modes
+
+The ruler supports two different rule evaluation modes, adaptable according to the specific needs of the user:
+
+- **Internal:** This is the default mode. When set, the ruler connects directly through its built-in querier to ingesters and store-gateways, and internally evaluates the rule expression to generate the resulting series.
+  In most use cases this mode is solvent enough. However, in those where the evaluation complexity is high enough, the latency can be increased up to the point of limiting the correct functioning of the component.
+
+- **Remote:** In this mode the ruler delegates rule expression evaluation to the query-frontend. In this way, we can leverage all the optimizations that this component offers us, such as [query sharding]({{< relref "../../query-sharding/index.md" >}}), allowing us to horizontally scale and drastically reducing evaluation times.<br/>
```

Comment:
We need the architecture diagram for this operational mode. @09jvilla who can we ask it?

Reply:
Marco, looks like you've already got your answer :) Song helped me put them together. Let me know if you need more help.

Reply:
I'm good, thanks! Work will be tracked by this issue: https://github.com/grafana/mimir/issues/1831